build.yaml

```diff
@@ -1 +1,8 @@
 projectType: extension
+additionalTestConfigs:
+  jdk11:
+    testJdkTool: TEMURIN-JDK11
+    mavenAdditionalArgs: -DruntimeVersion=4.4.0-20220221 -DadditionalRuntimeVersions=4.3.0-20220221 -DruntimeProduct=MULE_EE
+  jdk17:
+    testJdkTool: TEMURIN-JDK17
+    mavenAdditionalArgs: -DruntimeVersion=4.6.0-rc2 -DruntimeProduct=MULE_EE
```

Comment:
I don't think this build.yaml is correct I know there have been some changes to solve connectivity issues, but it seems to be testing jdk 11 only against Mule runtime 4.4.0 and 4.3.0, not against mule 4.5 and 4.6. I think we should consider those scenarios too. @elcoya any word on this?

Reply:
Please refer to the [build.yml](https://docs.google.com/document/d/1PBMVwFhtQ8XCCi1myTWz09NbvjWcUc8VrySqdeCvZBk/edit?pli=1#heading=h.ylfsbw9ketba) section in the playbook.
We should not set the runtime versions in a fixed way, rather than setting the minimum version to autodiscover